test/csuite/timestamp_abort/main.c

```diff
@@ -605,6 +605,11 @@ main(int argc, char *argv[])
 	    "cp -p WiredTigerLog.* ../%s.SAVE",
 	     home, home, home));
 	(void)system(buf);
+	testutil_check(__wt_snprintf(buf, sizeof(buf),
+	    "rm -rf ../%s.SAVE_FULL && mkdir ../%s.SAVE_FULL && "
+	    "cp -p * ../%s.SAVE_FULL",
+	     home, home, home));
+	(void)system(buf);
```

Comment:
Before merging this should be commented or ifdefed out.  I'm good leaving the code in, with a comment stating that it will be slow, tables are very, very large and many non-archived log files are created.  It will cause PR testing to go very slowly. That way if/when the next problem surfaces someone can just reinstate this for debugging rather than redoing it every time.  But it should not be part of the normal code.